### DISCUSSION TITLE
Why does the crosshair disappear when I use the digital pen? How can I make the cursor always visible when I'm using a digital pen?

### DISCUSSION THREAD

**Discussion by happyfunction** [excalidraw/excalidraw#8922] [Q&A]
When using the pen tool, the crosshair is always displayed when using the mouse, whether you are writing (pressing the left mouse button) or just moving the cursor. However, when using a digital pen and a graphics tablet, the cursor disappears when the digital pen touches the graphics tablet. When the digital pen leaves the graphics tablet, the cursor reappears. If you write relatively fast, the cursor will appear and disappear from time to time, and this kind of experience is not good.

**Comment by Mrazator** [comment]
Converted to an issue https://github.com/excalidraw/excalidraw/issues/8981
